dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/security/SecurityPage.java

```diff
@@ -68,58 +68,58 @@ public SecurityPage(RemoteWebDriver driver) {
 
     public <T extends SecurityPage.Tab> T goToTab(Class<T> tab) {
         if (tab == TenantPage.class) {
-            new WebDriverWait(driver, Duration.ofSeconds(60)).until(ExpectedConditions.urlContains("/security"));
```

Comment:
We should not remove this because some exceptions may occur when jumping to this page from other pages.

Reply:
The url is incorrect, I added 
```
new WebDriverWait(driver, Duration.ofSeconds(60)).until(ExpectedConditions.urlContains("/security/tenant-manage"));
``` 
at the below

Reply:
The first `WebDriverWait` is to make sure you're already on the page before you click on it. The second one is to make sure the clicking is finished.

Reply:
There already exist 
```
new WebDriverWait(driver, Duration.ofSeconds(60)).until(ExpectedConditions.elementToBeClickable(menuTenantManage));
```
This can make sure we already in security page.